store/interfaces.json

```diff
@@ -0,0 +1,5 @@
+{
+  "flixmedia": {
```

Comment:
What do you think of extending the interface `product-description`?

Example: `product-description.flixmedia`

That way we don't need to allow a new block in the store and I think it makes sense semantically.

Reply:
@klzns  the `product-description` is being used, how can we proceed in this case?

Reply:
You can use both. No problem.

blocks.json
```js
"store.product": {
    "children": [
      "product-description",
      "product-description.flixmedia",
    ]
}
```

Reply:
Ok @klzns!